python/ray/dag/dag_node.py

```diff
@@ -191,6 +193,8 @@ def experimental_compile(
                 are allowed to be sent to this DAG. Before submitting more requests,
                 the caller is responsible for calling ray.get to clear finished
                 in-flight requests.
+            _custom_nccl_group: The default custom NCCL group to be used for P2P NCCL
```

Comment:
Define "the default custom"?

Reply:
See above for definition and comment change.